x/auth/client/rest/query.go

```diff
@@ -17,7 +17,13 @@ import (
 	genutilrest "github.com/cosmos/cosmos-sdk/x/genutil/client/rest"
 )
 
-const unRegisteredConcreteTypeErr = "unregistered concrete type"
+// knownErrors are errors that happen on unmarshalling new proto-only txs using
+// amino. Also see https://github.com/cosmos/cosmos-sdk/issues/7639.
+var knownErrors = []string{
+	"unregistered interface",
+	"unregistered concrete type",
+	"wrong SignMode. Expected SIGN_MODE_LEGACY_AMINO_JSON, got SIGN_MODE_DIRECT",
```

Comment:
I'm not sure if this list is exhaustive, though it should handle most cases

Reply:
removed in https://github.com/cosmos/cosmos-sdk/pull/7992#issuecomment-732206733